packages/components/src/index.ts

```diff
@@ -299,6 +299,8 @@ import { SampleCreationTypeModal } from './internal/components/samples/SampleCre
 import { SamplesSelectionProvider } from './internal/components/samples/SamplesSelectionContextProvider';
 import { SampleAliquotDetailHeader } from './internal/components/samples/SampleAliquotDetailHeader';
 import { SampleAliquotsSummary } from './internal/components/samples/SampleAliquotsSummary';
+import { SampleAliquotsGridPanel } from './internal/components/samples/SampleAliquotsGridPanel';
```

Comment:
In addition to the changes you've made to `index.ts`, I would also recommend removing `getSampleAssayQueryConfigs` and `getSampleAliquotsQueryConfig` from being exported here. With your changes the only usages of these are within `@labkey/components` so they no longer need to be exported.

With this you'll need to update the import path of a couple of usages within the repository.

Reply:
Done (in patch)